### PR TITLE
snow-502 00374258 DEL character issue

### PR DESCRIPTION
**Pull request by SimbaGithub**
bug fix at ReusableChunkParser.cs and I created a test case for this.

### CODE REVIEW

File: Snowflake.Data.Tests/SFDelCharT.cs [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                    Assert.AreEqual(1, insertReader.RecordsAffected);
+                }
+                */
+                string selectCommand = "select * from deltest1";
+                cmd.CommandText = selectCommand;
+                cmd.CommandType = System.Data.CommandType.Text;
+
+                var sb = new StringBuilder();
+                var count = 0;
+                using (var sw = new System.IO.StreamWriter(@"D:\Snowflake\Log\test.txt", true, Encoding.UTF8, 4096))
```
Comment (sfc-gh-cshi) [inline Snowflake.Data.Tests/SFDelCharT.cs:117]: absolute path?

File: Snowflake.Data/Core/ReusableChunkParser.cs [hunk 2]
```diff
@@ -123,6 +123,9 @@ public async Task ParseChunk(IResultChunk chunk)
                                     break;
                                 case -1:
                                     throw new SnowflakeDbException(SFError.INTERNAL_ERROR, $"Unexpected end of stream in escape sequence");
+                                default:
```
Comment (sfc-gh-cshi) [inline Snowflake.Data/Core/ReusableChunkParser.cs:126]: Do we need to handle other escape case as well? Make sure we are good with all these cases: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/strings/#string-escape-sequences

Comment (SimbaGithub) [reply Snowflake.Data/Core/ReusableChunkParser.cs:126]: it already cover all escape cases.

File: Snowflake.Data.Tests/SFDelCharT.cs (hunk 1 above)
Comment (SimbaGithub) [reply Snowflake.Data.Tests/SFDelCharT.cs]: I updated

**Review by sfc-gh-cshi — APPROVED** [review]
LGTM. comments on test range 35000 and 4000 should be helpful

**Pull request merged**